crates/egui/src/text_selection/visuals.rs

```diff
@@ -73,3 +73,44 @@ pub fn paint_cursor(painter: &Painter, visuals: &Visuals, cursor_rect: Rect) {
         );
     }
 }
+
+/// Paint text cursor.
+pub fn paint_text_cursor(
```

Comment:
`fn paint_text_cursor` and `fn paint_cursor` have very similar names, but one does blinking, and one does not. We should document that. `fn paint_cursor` should probably be made non-`pub` too